test/TestPrisonersDilemma.js

```diff
@@ -175,4 +175,22 @@ contract('PrisonersDilemma', async (accounts) => {
         //Test if a player can continue to play after there is a winner
         await expectThrow(instance.playerChoose(CHOICES["Take"], { from: accounts[0] }))
     });
+
+    it("Gas Analysis", async() => {
+        var estimate = 0;
+        var receipt = 0;
+
+        console.log(`Web 3 Api Version: ${ web3.version.api }`);
+        console.log(`Web 3 Ethreum Version: ${ web3.version.ethereum }`);
+
+        // Create
+        receipt = await web3.eth.getTransactionReceipt(instance.transactionHash);
```

Comment:
Not confident I did this correctly, especially since this is after the contract is deployed.

Reply:
I think the other way to do it would be to get the total number of bytes from the abi and then estimate gas usage.